Firestore/Example/Tests/Integration/API/FIRServerTimestampTests.m

```diff
@@ -76,7 +89,9 @@ - (void)tearDown {
   [super tearDown];
 }
 
-// Returns the expected data, with an arbitrary timestamp substituted in.
+#pragma mark - Test Helpers
+
+/** Returns the expected data, with the specified timestamp substituted in. */
 - (NSDictionary *)expectedDataWithTimestamp:(id _Nullable)timestamp {
```

Comment:
Can't the type of the argument be `nullable id`?

Reply:
Changed to nullable id.